bots/john/assets/var/jokes.json

```diff
@@ -1,86 +0,0 @@
-[
-   {
-      "joke":"Did you hear about the guy whose whole left side was cut off? He's all right now."
```

Comment:
Actually I am not sure. I didn't remove this, I just renamed it (like I did everything else). I don't know why it is showing up as a removal.